### ISSUE TITLE
[Bug] The model and loaded state dict do not match exactly

### ISSUE THREAD

**Issue by XinDing5** [open-mmlab/mmocr#1878]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmocr/issues) and [Discussions](https://github.com/open-mmlab/mmocr/discussions) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (0.x)](https://github.com/open-mmlab/mmocr) or [latest version (1.x)](https://github.com/open-mmlab/mmocr/tree/dev-1.x).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

main branch https://github.com/open-mmlab/mmocr

### Environment

sys.platform: linux
Python: 3.8.13 | packaged by conda-forge | (default, Mar 25 2022, 06:04:10) [GCC 10.3.0]
CUDA available: True
numpy_random_seed: 2147483648
GPU 0,1: Tesla P100-PCIE-16GB
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 11.7, V11.7.99
GCC: gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
PyTorch: 1.13.0a0+340c412
PyTorch compiling details: PyTorch built with:
  - GCC 9.4
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.4 Product Build 20200917 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v2.6.0 (Git Hash N/A)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - LAPACK is enabled (usually provided by MKL)
  - NNPACK is enabled
  - CPU capability usage: AVX512
  - CUDA Runtime 11.7
  - NVCC architecture flags: -gencode;arch=compute_52,code=sm_52;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86;-gencode;arch=compute_86,code=compute_86
  - CuDNN 8.4.1  (built against CUDA 11.6)
  - Magma 2.6.2
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.7, CUDNN_VERSION=8.4.1, CXX_COMPILER=/usr/bin/c++, CXX_FLAGS=-fno-gnu-unique -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -DEDGE_PROFILER_USE_KINETO -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Werror=cast-function-type -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.13.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=OFF, USE_MPI=ON, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_ROCM=OFF, 

TorchVision: 0.13.0a0
OpenCV: 4.2.0
MMEngine: 0.7.2
MMOCR: 1.0.0+d7c59f3

### Reproduces the problem - code sample

Here is my config file:

_base_ = [
    '_base_dbnet_resnet18_fpnc.py',
    '../_base_/datasets/icdar2015.py',
    '../_base_/datasets/rects.py',
    '../_base_/default_runtime.py',
    '../_base_/schedules/schedule_sgd_1200e.py',
]

train_list = [_base_.icdar2015_textdet_train, _base_.rects_textdet_train]
test_list = [_base_.icdar2015_textdet_test, _base_.rects_textdet_test]

train_dataset = dict(type='ConcatDataset', datasets=train_list, pipeline=_base_.train_pipeline)
test_dataset = dict(type='ConcatDataset', datasets=test_list, pipeline=_base_.test_pipeline)

train_dataloader = dict(
    batch_size=32,
    num_workers=8,
    persistent_workers=True,
    sampler=dict(type='DefaultSampler', shuffle=True),
    dataset=train_dataset)

val_dataloader = dict(
    batch_size=1,
    num_workers=4,
    persistent_workers=True,
    sampler=dict(type='DefaultSampler', shuffle=False),
    dataset=test_dataset)

test_dataloader = val_dataloader

load_from = '/dfs/data/mmocr/work_dirs/models/dbnet_resnet18_fpnc_100k_synthtext-2e9bf392.pth'
resume = True
train_cfg = dict(type='EpochBasedTrainLoop', max_epochs=3000, val_interval=20)
optim_wrapper = dict(
    optimizer=dict(type='SGD', lr=0.0007, momentum=0.9, weight_decay=0.00001))

auto_scale_lr = dict(base_batch_size=32)

### Reproduces the problem - command or script

Here is my command:

tools/dist_train.sh configs/textdet/dbnet/dbnet_resnet18_fpnc_1200e_rects_icdar2015.py 2

### Reproduces the problem - error message

04/25 19:08:10 - mmengine - INFO - load model from: torchvision://resnet18
04/25 19:08:10 - mmengine - INFO - Loads checkpoint by torchvision backend from path: torchvision://resnet18
04/25 19:08:10 - mmengine - WARNING - The model and loaded state dict do not match exactly

unexpected key in source state_dict: fc.weight, fc.bias

Loads checkpoint by local backend from path: work_dirs/models/dbnet_resnet18_fpnc_100k_synthtext-2e9bf392.pth
Loads checkpoint by local backend from path: work_dirs/models/dbnet_resnet18_fpnc_100k_synthtext-2e9bf392.pth

### Additional information

When I try to train DBnet from the provided pre-trained Syntext DB model, it says the model and loaded state dict do not match exactly. The training does not start. Could you please help to have a look?

**Comment by XinDing5** [comment]
> See FAQ Q1: https://mmocr.readthedocs.io/en/dev-1.x/get_started/faq.html

But the training cannot start even if I ignore the warning...Why is that?

**Comment by gaotongxiao** [comment]
You can monitor your CPU and/or GPU usage. Training had probably started but was so slow that you did not even reach the first log interval.

**Comment by XinDing5** [comment]
> You can monitor your CPU and/or GPU usage. Training had probably started but was so slow that you did not even reach the first log interval.

Thanks for reply, but I believe it is not the case for me .... The process stopped after printing the above output...
